crypto/asn1/a_time.c

```diff
@@ -561,3 +561,44 @@ int ASN1_TIME_compare(const ASN1_TIME *a, const ASN1_TIME *b)
         return -1;
     return 0;
 }
+
+/*
+ * tweak for Windows
+ */
+#ifdef WIN32
+# define timezone _timezone
+#endif
+
+time_t *asn1_string_to_time_t(char *asn1_string)
```

Comment:
Why do you use pointer to time_t as a return value when time_t is just a scalar value?

Reply:
Simply because `X509_time_adj` needs a pointer to time_t as a parameter. Of course it's better to change `asn1_string_to_time_t` to return a time_t and change calling of `do_updatedb`. I've already changed this locally, but I will wait with pushing it until I get the results from AppVeyor from the last push.